lib/iris/cube.py

```diff
@@ -1654,14 +1654,14 @@ def lazy_data(self):
             A lazy array, representing the Cube data array.
 
         """
-        if self._numpy_array is not None:
```

Comment:
We've defined a method to make this check, so let's use it rather than repeating the same code.